docs/guides/zookeeper/update-version/overview.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Updating ZooKeeper Overview
+menu:
+  docs_{{ .version }}:
+    identifier: zk-update-version-overview
+    name: Overview
+    parent: zk-update-version
+    weight: 10
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+> New to KubeDB? Please start [here](/docs/README.md).
+
+# updating ZooKeeper version Overview
+
+This guide will give you an overview on how KubeDB Ops-manager operator update the version of `ZooKeeper` database.
+
+## Before You Begin
+
+- You should be familiar with the following `KubeDB` concepts:
+    - [ZooKeeper](/docs/guides/zookeeper/concepts/zookeeper.md)
+    - [ZooKeeperOpsRequest](/docs/guides/zookeeper/concepts/opsrequest.md)
+
+## How update version Process Works
+
+The following diagram shows how KubeDB Ops-manager operator used to update the version of `ZooKeeper`. Open the image in a new tab to see the enlarged version.
+
+<figure align="center">
+  <img alt="updating Process of ZooKeeper" src="/docs/images/day-2-operation/zookeeper/zk-version-update.svg">
+<figcaption align="center">Fig: updating Process of ZooKeeper</figcaption>
+</figure>
+
```

Comment:
In the picture the word used is "upgrade". We want to use the word "Update" more often.